src/components/Header/index.tsx

```diff
@@ -98,3 +142,10 @@ const Title = styled(View)`
   flexDirection: column;
   alignItems: center;
 `;
+
+const mapStateToProps = state => ({ auth: state.auth, chat: state.chat });
```

Comment:
I'd make it as this
const mapStateToProps = ({ auth, chat}) => ({ auth, chat});